pkg/crypto/api.go

```diff
@@ -1,20 +1,15 @@
-/*
-Copyright IBM Corp. All Rights Reserved.
-
-SPDX-License-Identifier: Apache-2.0
-*/
-
```

Comment:
Keep it?

Reply:
This file is also very recent, the content already created outside of IBM, so it can stay removed.

Reply:
In fact, I've slimmed it down even more (first I had it on a different branch, but in fact it also falls into "module generalization", as it generalizes the interface of what used to be the crypto module). Pushing the new version here.

Reply:
[This file is almost the same](https://github.com/filecoin-project/mir/compare/10658fbe24ca2bb33ac3151daf7d6f610f4a16e2...matejpavlovic:general-modules#diff-3452a5463a9ec45fcb2cd44aaee2013b82bdba6c182cd2d1da6c159da4bb5085).

Reply:
Wow ok, I really thought I wrote it after leaving IBM. Let's keep it then.

Reply:
Btw, apparently, formally, you should indicate that the files have been changed.

> You must cause any modified files to carry prominent notices stating that You changed the files.